app/views/declaration/summary/normal_summary_page.scala.html

```diff
@@ -44,7 +44,11 @@
     paragraphBody: paragraphBody
 )
 
-@(backLink: Call, errors: Seq[FormError] = Seq.empty[FormError], maybeContinuePlaceholder: Option[String] = None)(implicit request: JourneyRequest[_], messages: Messages, appConfig: AppConfig)
+@(mode: Mode,
```

Comment:
Why is the mode is reintroduced for summary page?

Reply:
Because all sections in this page, like `@references_section`, `@parties_section`, ..., still need to be called with `mode` as first parameter, as they are still passing it to every internal route they contain.

My guess then was that it was safer to change them once we completely wipe out all modes (with another ticket).